bc_obps/reporting/schema/report_final_review.py

```diff
@@ -372,6 +372,11 @@ class BaseReportVersionSchema(ModelSchema):
     report_new_entrant: List[ReportNewEntrantSchema] = []
     operation_emission_summary: Optional[EmissionSummarySchemaOut] = None
     is_supplementary_report: Optional[bool] = None
+    reporting_year: Optional[int] = None
+
+    @staticmethod
+    def resolve_reporting_year(obj: ReportVersion) -> Optional[int]:
+        return obj.report.reporting_year.reporting_year if obj.report and obj.report.reporting_year else None
```

Comment:
Errors shouldn't be silenced. A report version should always have a report and a reporting_year attached

```suggestion
    @staticmethod
    def resolve_reporting_year(obj: ReportVersion) -> Optional[int]:
        return obj.report.reporting_year.reporting_year
```